waiter/test/waiter/core_test.clj

```diff
@@ -191,16 +191,13 @@
                         (if (instance? Throwable result)
                           (throw result)
                           result))))
-        configuration {:daemons {:populate-maintainer-chan! {:query-state-fn (constantly {})}
-                                 :router-state-maintainer {:maintainer {:notify-instance-killed-fn (constantly {})}}}
-                       :routines {:peers-acknowledged-eject-requests-fn (constantly true)
-                                  :allowed-to-manage-service?-fn (constantly true)
-                                  :service-id->service-description-fn (constantly {})}
-                       :scheduler {:scheduler scheduler}
-                       :state {:scaling-timeout-config scaling-timeout-config
-                               :scheduler-interactions-thread-pool scheduler-interactions-thread-pool}
-                       :wrap-ignore-disabled-auth-fn utils/wrap-identity
-                       :wrap-secure-request-fn utils/wrap-identity}
+        notify-instance-killed-fn (constantly {})
```

Comment:
do we normally name functions `-fn`?

Reply:
Yes, we generally add the `-fn` suffix for local variables representing function, you can also it above with `wrap-secure-request-fn`. 
But there isn't a lot consistency around it in the codebase and there are other occurrences without that suffix :(